packages/scripts/src/tasks/index.ts

```diff
@@ -0,0 +1,27 @@
+import android from "./providers/android";
+import appData from "./providers/appData";
+import file from "./providers/file";
+import gdrive from "./providers/gdrive";
+import template from "./providers/template";
+import translate from "./providers/translate";
+import workflow from "./providers/workflow";
+
+/**
+ * Tasks are thin wrappers around functions and commands, to support inclusion as part of
+ * chained workflows
+ *
+ * TODO - naming conventions should be tidied so that commands and tasks are consistend
+ */
```

Comment:
This is a list of all tasks currently either defined or under construction, individual scripts can be found in files that follow